tests/Tester.py

```diff
@@ -148,15 +148,15 @@ def setUpClass(cls):
         os.environ["ENABLE_SKIP_FAULTY"] = 'True'  # Always test with skipping faulty enabled
 
     def snippet_already_tested_with_current_hedy_version(self, test_hash):
-        try:
-            total_hash_incl_the_hedy_language = create_hash(get_hedy_source_hash(), test_hash)
-            if total_hash_incl_the_hedy_language is None:
-                return False
-            filename = get_hash_filename(total_hash_incl_the_hedy_language)
-            already_successful = os.path.isfile(filename)
-            return already_successful
-        except UnicodeEncodeError:  # some tests (generated by Hypothesis) can't be hashed
-            return False
+        # try:
```

Comment:
Was this an unintended change?

Reply:
Yes, definitely. Thanks for pointing it out. The Hypothesis test for level 1 was failing locally and I wanted to see whether it succeeds in Github because of caching. So I accidentally left this in.